plugins/modules/cloud/docker/docker_stack_facts.py

```diff
@@ -0,0 +1 @@
+docker_stack_info.py
```

Comment:
Please remove this. We do not add new _facts modules that should not be _facts modules, and this is one of them.

(You must add a symlink from plugins/modules/ to the module though.)

Reply:
Removed. Thanks